Libraries/PeriphDrivers/Source/UART/uart_revb_regs.h

```diff
@@ -265,6 +268,9 @@ typedef struct {
  #define MXC_F_UART_REVB_INT_FL_RX_THD_POS              4 /**< INT_FL_RX_THD Position */
  #define MXC_F_UART_REVB_INT_FL_RX_THD                  ((uint32_t)(0x1UL << MXC_F_UART_REVB_INT_FL_RX_THD_POS)) /**< INT_FL_RX_THD Mask */
 
+ #define MXC_F_UART_REVB_INT_FL_TX_OR_POS               5 /**< INT_FL_TX_OR Position */
+ #define MXC_F_UART_REVB_INT_FL_TX_OR                   ((uint32_t)(0x1UL << MXC_F_UART_REVB_INT_FL_TX_OR_POS)) /**< INT_FL_TX_OR Mask */
```

Comment:
Hmm, I guess I didn't regenerate the UART RevB register files after adding the UART one byte remaining interrupt field. The SVD and the ME30 user guide use the `TX_OB` name:

```
 #define MXC_F_UART_REVB_INT_EN_TX_OB_POS               5 /**< INT_EN_TX_OB Position */
 #define MXC_F_UART_REVB_INT_EN_TX_OB                   ((uint32_t)(0x1UL << MXC_F_UART_REVB_INT_EN_TX_OB_POS)) /**< INT_EN_TX_OB Mask */
```

```
 #define MXC_F_UART_REVB_INT_FL_TX_OB_POS               5 /**< INT_FL_TX_OB Position */
 #define MXC_F_UART_REVB_INT_FL_TX_OB                   ((uint32_t)(0x1UL << MXC_F_UART_REVB_INT_FL_TX_OB_POS)) /**< INT_FL_TX_OB Mask */
```